supabase/migrations/20260611100000_realtime_notification_rls.sql

```
@@ -0,0 +1,43 @@
+-- orders: REPLICA IDENTITY FULL (UPDATE payload에 old 레코드 포함)
+-- supabase_realtime publication에 양 테이블 추가
+
+-- 1. payment_events: authenticated 사용자 SELECT 권한 부여
+GRANT SELECT ON TABLE payment_events TO authenticated;
+
+-- 2. payment_events: 판매자 SELECT RLS 정책 (자기 가게 이벤트만 조회 가능)
+CREATE POLICY "seller_own_store_events"
+  ON payment_events FOR SELECT
+  USING (
+    store_id IN (
+      SELECT id FROM stores WHERE user_id = auth.uid()
```

SQLFluff (4.2.1)

[error] The 'WHERE' keyword should always start a new line.

(LT14)